core/src/main/scala/cats/data/NonEmptyList.scala

```diff
@@ -149,6 +180,19 @@ final case class NonEmptyList[+A](head: A, tail: List[A]) extends NonEmptyCollec
   def append[AA >: A](a: AA): NonEmptyList[AA] =
     NonEmptyList(head, tail :+ a)
 
+  /**
+   * Alias for [[concat]]
+   *
+   * {{{
+   * scala> import cats.data.NonEmptyList
+   * scala> val nel = NonEmptyList.of(1, 2, 3)
+   * scala> nel.appendList(List(4, 5))
+   * res0: cats.data.NonEmptyList[Int] = NonEmptyList(1, 2, 3, 4, 5)
+   * }}}
+   */
+  def appendList[AA >: A](other: List[AA]): NonEmptyList[AA] =
+    concat(other)
```

Comment:
Do you mind adding a test for this one as well? Thanks!

Reply:
It's already in https://github.com/typelevel/cats/pull/4267/files#diff-984617a3edbb35fcc2aebfff12d3bebcf652a04fcfc327905b6cc76dde536efdR388.

Reply:
Ah it is, sorry I missed that!